invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/index.js

```diff
@@ -45,6 +50,13 @@ class DashboardTabs extends Component {
   constructor(props) {
     super(props);
     const activeTabName = rootElement.dataset[_camelCase("active-tab-name")];
+    const communitiesEnabled =
+      rootElement.dataset[_camelCase("communities-enabled")] === "True";
+
+    TAB_PANES = communitiesEnabled
+      ? DEFAULT_TAB_PANES.concat(COMMUNITY_TAB_PANES)
+      : DEFAULT_TAB_PANES;
+
     const routes = TAB_PANES.map((pane) => pane.pathname);
```

Comment:
it would be much more readable if you do without splitting the tabs arrays above
```suggestion
    const communitiesEnabled =
      rootElement.dataset[_camelCase("communities-enabled")] === "True";

    TAB_PANES = communitiesEnables ? TAB_PANES : [TAB_PANES[0]]

    const routes = TAB_PANES.map((pane) => pane.pathname);
```